### PR TITLE
Add bencode

### PR DESCRIPTION
**Pull request by fraidev**

### CODE REVIEW

File: ocaml/default.nix [hunk 1]
```diff
@@ -136,6 +136,16 @@ with oself;
     };
   };
 
+  bencode = osuper.buildDunePackage {
```
Comment (anmonteiro) [inline ocaml/default.nix:139]:
```suggestion
  bencode = buildDunePackage {
```

File: ocaml/default.nix [hunk 2]
```diff
@@ -136,6 +136,16 @@ with oself;
...
+    pname = "bencode";
+    version = "2.0";
+    src = builtins.fetchurl {
+      url = "https://github.com/rgrinberg/bencode/archive/2.0.tar.gz";
+      sha256 =
+        "233eae0817126e9c4a781bf1329d834672b77954e6b983a4d0a298d1e2ff0756";
```
Comment (anmonteiro) [inline ocaml/default.nix:145]:
```suggestion
      sha256 = "233eae0817126e9c4a781bf1329d834672b77954e6b983a4d0a298d1e2ff0756";
```

File: ocaml/default.nix (hunk 1 above)
Comment (fraidev) [reply ocaml/default.nix]: done

File: ocaml/default.nix (hunk 2 above)
Comment (fraidev) [reply ocaml/default.nix]: done

**Comment by anmonteiro** [comment]
see https://github.com/nix-ocaml/nix-overlays/pull/556

**Pull request closed without merging**